examples/magento-graphcms/pages/switch-stores.tsx

```diff
@@ -28,7 +28,11 @@ function StoresIndexPage({ availableStores }: Props) {
 
   return (
     <>
-      <PageMeta title={i18n._(/* i18n */ 'Switch stores')} metaRobots={['noindex']} />
+      <PageMeta
+        title={i18n._(/* i18n */ 'Switch stores')}
+        metaDescription={i18n._(/* i18n */ 'Switch stores')}
```

Comment:
Page is not indexable, metaDescription is useless. Adding this change creates useless churn